src/main/java/se/yolean/kafka/keyvalue/ConsumerAtLeastOnce.java

```diff
@@ -67,8 +67,8 @@ public enum Stage {
 
   final Logger logger = LoggerFactory.getLogger(this.getClass());
 
-  @ConfigProperty(name = "kkc.assignments.timeout", defaultValue="90s")
-  private Duration assignmentsTimeout;
+  @ConfigProperty(name = "kkc.assignments.timeout", defaultValue="90")
```

Comment:
486be0b40ed38bdaaeb13df3c3143fe653e35c5b is breaking if we have used the config anywhere. I think it would be worthwhile resurrecting Duration parsing.